deploy/aws/network.tf

```diff
@@ -157,17 +156,17 @@ resource "aws_security_group" "qed" {
     protocol    = "tcp"
     cidr_blocks = ["${chomp(data.http.ip.body)}/32"]
   }
-  ingress  {
-      from_port    = 0
-      to_port      = 65535
-      protocol     = "tcp"
-      self         = true
+  ingress {
```

Comment:
Why do we need this? Do we have unidentified communications?

Reply:
This rule only allows communication between the machines which has the security group attached. Of course, we can elaborate more restrictive security groups to allow communications only between the mandatory service ports.

Reply:
I think we should, even if it is a bit of work. That would allow us to be sure what we need to be explicity allowed for the whole system to work.

Reply:
I've created an issue #169  to address this.